packages/vow/src/rejection-tracker.js

```diff
@@ -25,73 +25,73 @@ const makeUnhandledRejection = (rejected, upgradedPK) => {
   };
 };
 
-const VowCapShape = M.remotable('VowCap');
-const VowRejectionTrackerKitI = {
-  public: M.interface('VowRejectionTracker', {
-    handle: M.call(VowCapShape).returns(),
-    reject: M.call(VowCapShape, M.raw(), M.promise()).returns(),
-  }),
-  upgradeRejectionWatcher: M.interface('UpgradeRejectionWatcher', {
-    onRejected: M.call(M.raw(), M.raw()).returns(),
-  }),
-};
-
 /** @param {Zone} zone */
 export const prepareVowRejectionTracker = zone => {
```

Comment:
Shouldn't this be named `provide` now that it returns a singleton?

Reply:
AFAIK, `prepare*` is the conventional name for a function that is parameterized by a zone, and `provide*` is for something idempotent, which this is not.